e2e_tests/integration/index.spec.js

```diff
@@ -108,6 +108,25 @@ describe('Neo4j Browser', () => {
       .get('[data-test-id="sidebarMetaItem"]', { timeout: 30000 })
       .should('have.length', 18)
   })
+  it('will add parameter using `:param` command', () => {
+    // add cypher evalutated param function
+    const command = ':param foo => 1 + 1'
+    cy.get(Editor).type(command, { force: true })
```

Comment:
Nice with e2e test! 
You could use `cy.executeCommand(command)` here instead to clean up a bit.